.github/workflows/e2e.yml

```diff
@@ -11,7 +11,7 @@ env:
   FORCE_COLOR: 3
   PIP_DISABLE_PIP_VERSION_CHECK: 1
   # https://pypi.org/project/pytest-rerunfailures/
-  PYTEST_ADDOPTS: '--reruns 1'
+  # PYTEST_ADDOPTS: '--reruns 1'
```

Comment:
what does this env var do?

Reply:
It reruns failing tests. I removed it to unmask tst flakiness and to ease debugging tests

Reply:
Added TODO to roll it back or to remove it